### PR TITLE
Update Dockerfile for new cpuminer location

### PR DESCRIPTION
**Pull request by steveswinsburg**
Old location is no longer maintained.

### CODE REVIEW

**Comment by kannix** [comment]
Where did you find the url for the new repo?

Because the old repo says that the updated url is 
https://github.com/genesismining/sgminer-gm
instead of
https://github.com/OhGodAPet/cpuminer-multi

all in all both repos look like they are abandoned 😞

File: Dockerfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ RUN		apt-get update -qq && apt-get install -qqy \
   make \
   build-essential
 
-RUN		git clone https://github.com/wolf9466/cpuminer-multi
```
Comment (steveswinsburg) [inline Dockerfile]: When you go to the wolf4966 repo, it redirects to the OhGodAPet one. They are functionally equivalent.

Interesting about the new new code. I haven't used it yet but will try to get a Docker file and image up for that one, it is a bit different.

**Comment by steveswinsburg** [comment]
Sorry I submitted that as a review instead of comment. Trying to do this on the bus 😑

**Comment by kannix** [comment]
Closing this because I switched to xmrig 😄

**Pull request closed without merging**